assets/js/googlesitekit/modules/datastore/constants.js

```diff
@@ -1,5 +1,5 @@
 /**
- * core/site data store: constants.
+ * Core site data store: constants.
```

Comment:
This is actually incorrect here anyway, let's fix it:
```suggestion
 * `core/modules` data store: constants.
```